logs/audit.log

```diff
@@ -0,0 +1,268 @@
+{"level":"info","username":"mrturkmen","time":"2019-07-22T08:49:22Z","message":"signup user"}
```

Comment:
Prefer to remove this file from repo (I know it is already ignored by `.gitignore`, but this file still remains in PR).

If you do something like this, you keep the file locally on your machine. but remove it from git (need to commit afterwards):
`git rm --cached logs/audit.log`

Reply:
ok